### PR TITLE
Upgrade to sdk rc9

### LINKED ISSUE #90: Upgrade to SDK RC9

### PR DESCRIPTION
**Pull request by smallhive**
close #90

### CODE REVIEW

File: cmd/neofs-rest-gw/integration_test.go [hunk 1]
```diff
@@ -81,7 +81,6 @@ func runLocalTests(ctx context.Context, t *testing.T, key *keys.PrivateKey) {
 func runTestInContainer(rootCtx context.Context, t *testing.T, key *keys.PrivateKey) {
 	aioImage := "nspccdev/neofs-aio-testcontainer:"
 	versions := []string{
-		"0.29.0",
 		"0.30.0",
```
Comment (roman-khimov) [inline cmd/neofs-rest-gw/integration_test.go:84]: Let's drop this one too.

**Review by roman-khimov — COMMENTED** [review]
How about testing with `nspccdev/neofs-aio` of current versions?

File: cmd/neofs-rest-gw/integration_test.go (hunk 1 above)
Comment (smallhive) [reply cmd/neofs-rest-gw/integration_test.go]: `0.30.0` removed

**Comment by smallhive** [comment]
> How about testing with `nspccdev/neofs-aio` of current versions?

Tried to use `nspccdev/neofs-aio` instead of `nspccdev/neofs-aio-testcontainer` and failed. It hadn't started correct even one time. I spent some time to debug and left as is. We may investigate this in separate issue

**Review by roman-khimov — APPROVED** [review]
OK, #92.

**Pull request merged**